ts/components/messages/MessageAgenda.tsx

```diff
@@ -236,12 +266,73 @@ const FakeItemComponent = (
  */
 class MessageAgenda extends React.PureComponent<Props, State> {
   private sectionListRef = React.createRef<any>();
+  private overScrollAnim: Animated.Value;
+
+  public componentDidMount() {
+    this.state.scrollY.addListener(value => this.handleScroll(value));
+  }
+
+  public componentWillUnmount() {
+    this.state.scrollY.removeAllListeners();
+  }
+
+  public handleRelease() {
+    if (pot.isLoading(this.state.potAnim) && !pot.isSome(this.state.potAnim)) {
+      // start animation
+      this.setState({ potAnim: pot.some(this.state.potAnim) });
+      this.animateOverScroll();
+    }
+  }
+
+  private handleScroll(pullDownDistance: any) {
+    if (pullDownDistance.value <= MIN_PULLDOWN_DISTANCE) {
+      return this.setState({ potAnim: pot.toLoading(this.state.potAnim) });
```

Comment:
```suggestion
     if (pot.isNone(this.state.potAnim)) {
        return this.setState({ potAnim: pot.noneLoading });
      }
```
if you are scrolling under the MIN_PULLDOWN_DISTANCE and the pot is none then change the state in loading (the animation is elegible to start)